README.md

```diff
@@ -449,6 +449,19 @@ FOO=BAR
 BAR=FOO
 ```
 
+Control comments also split lines (this is done to make the linter logic more predictable):
```

Comment:
Please add this information to this [page](https://github.com/dotenv-linter/dotenv-linter/blob/master/docs/checks/unordered_key.md) too.

Reply:
Done